resources/js/requests/components/RequestsListing.vue

```diff
@@ -113,6 +112,32 @@ export default {
       ]
     };
   },
+  beforeCreate() {	
+    //switch through all status
+    switch (Processmaker.status) {	
+      // if there is no status, meaning its on my requests, We should only show the in progress status
+      case "":	
```

Comment:
Is this an empty case or the default case?

Reply:
Can we dry up this functionality?

Reply:
this is there is an empty case